src/main/java/duke/MainUi.java

```diff
@@ -0,0 +1,21 @@
+package duke;
```

Comment:
Perhaps you can consider having a separate sub-package (duke.ui) for all your Ui to improve readability.

Reply:
Observant enough. 😄  I did the rest of the package structure in such a way that inheriting class under parent class is of same sub-package. However, forgotten about MainUi extends Ui.